modules/solid_mechanics/src/userobjects/CrackFrontDefinition.C

```diff
@@ -22,6 +22,7 @@ void addCrackFrontDefinitionParams(InputParameters& params)
 {
   MooseEnum direction_method("CrackDirectionVector CrackMouth CurvedCrackFront");
   MooseEnum end_direction_method("NoSpecialTreatment CrackDirectionVector", "NoSpecialTreatment");
+  params.addParam<std::vector<RealVectorValue> >("crack_front_points","Set of points to define crack front");
```

Comment:
MOOSE supports params of type std::vector<Points> and has for a long time. There aren't any advantages of using points over RealVectorValue today, but in the future we may have different checking mechanisms for Points.